content/en/case-studies/_index.html

```diff
@@ -1,5 +1,6 @@
 ---
 title: Case Studies
+linkTitle: View all case studies 
```

Comment:
The `title` and `linkTitle` should match:

> linkTitle: Case Studies

Reply:
Fixed.